### ISSUE TITLE
How to set float formatting using config preferences

### ISSUE THREAD

**Issue by sandsmichael** [dmnfarrell/pandastable#214]
Is this functionality supported? I see that there is a decimal place precision option, but I want to comma separate the numbers in my table while still retaining the ability to sort by column values on the table itself. Is this possible?

I do not see an example of this here:
https://pandastable.readthedocs.io/en/latest/examples.html#set-preferences

**Comment by dmnfarrell** [comment]
You mean you want thousands shown with a comma?

**Comment by sandsmichael** [comment]
Yes. Ideally, 1000000.0 would appear in the table as 1,000,000.0 but the columns of the table itself would remain sortable. Currently, if I string format the numbers before populating them into the table they obviously sort as strings instead of as floats.

**Comment by dmnfarrell** [comment]
There is now a thousands sep option in the preferences. See if this works.
